server_integration_test.go

```diff
@@ -71,8 +71,8 @@ var sftpServerDebugStream = ioutil.Discard
 var sftpClientDebugStream = ioutil.Discard
 
 const (
-	GOLANG_SFTP  = true
-	OPENSSH_SFTP = false
+	GolangSFTP  = true
+	OpenSSHSFTP = false
```

Comment:
More public consts... in this case it'd be fine to add the new ones, but we can't get rid of the old ones wihtout breaking compatibility. Aliases would work again.

Reply:
we need aliases for server_integration_test?

Reply:
Opps. No. Tests don't need to worry about this. Was rushing a bit I guess. Thanks.